React/Views/RCTSwitchManager.m

```diff
@@ -18,6 +18,8 @@ @implementation RCTSwitchManager
 
 RCT_EXPORT_MODULE()
 
+#if !TARGET_OS_TV
```

Comment:
Why not gate the whole module? This will make it so that every time you use Switch it'll render as a View instead